src/shipit/api/shipit_api/api.py

```diff
@@ -312,11 +317,62 @@ def update_release_status(name, body):
     return release
 
 
-@mozilla_accept_token()
 def get_phase_signoff(name, phase):
-    pass
+    session = g.db.session
+    try:
+        phase = session.query(Phase) \
+            .filter(Release.id == Phase.release_id) \
+            .filter(Release.name == name) \
+            .filter(Phase.name == phase).one()
+        signoffs = [s.json for s in phase.signoffs]
+        return dict(signoffs=signoffs)
+    except NoResultFound:
+        abort(404)
 
 
 @mozilla_accept_token()
-def phase_signoff(name, phase, body):
-    pass
+def phase_signoff(name, phase, uid):
+    session = g.db.session
+    try:
```

Comment:
maybe it would be nice to not nest this `try:` statements and only _wrap_ sections of code where you expect errors.

Reply:
Sure, let me redo it.

Reply:
Sure, let me redo it.